app/models/account_invitation.rb

```diff
@@ -1,4 +1,11 @@
 class AccountInvitation < ApplicationRecord
   belongs_to :user
   belongs_to :account
+
+  scope :accepted_by, lambda { |current_user|
+                        where(email: current_user.email).where.not(accepted_at: nil)
+                      }
+  scope :unaccepted_for, lambda { |current_user|
+                           where(email: current_user.email).where(accepted_at: nil)
+                         }
```

Comment:
```suggestion

  scope :for, -> (user) { |user| where(email: user.email) }
  scope :accepted,  -> { where.not(accepted_at: nil) }
  scope :unaccepted, -> { where(accepted_at: nil) }
```

Separate scopes give more flexibility and can be combined in different combinations.